RecoLocalMuon/GEMSegment/plugins/ME0SegAlgoRU.cc

```diff
@@ -29,14 +29,16 @@ ME0SegAlgoRU::ME0SegAlgoRU(const edm::ParameterSet& ps)
 
 	allowWideSegments = ps.getParameter<bool>("allowWideSegments");
 	doCollisions = ps.getParameter<bool>("doCollisions");
+
 	stdParameters.maxChi2Additional = ps.getParameter<double>("maxChi2Additional");
 	stdParameters.maxChi2Prune      = ps.getParameter<double>("maxChi2Prune"     );
 	stdParameters.maxChi2GoodSeg    = ps.getParameter<double>("maxChi2GoodSeg"   ) ;
 	stdParameters.maxPhiSeeds       = ps.getParameter<double>("maxPhiSeeds"      ) ;
 	stdParameters.maxPhiAdditional  = ps.getParameter<double>("maxPhiAdditional" );
 	stdParameters.maxETASeeds       = ps.getParameter<double>("maxETASeeds"      );
 	stdParameters.maxTOFDiff        = ps.getParameter<double>("maxTOFDiff"      );
-	stdParameters.minNumberOfHits        = ps.getParameter<unsigned int>("minNumberOfHits"      );
+	stdParameters.requireCentralBX  = ps.getParameter<bool>("requireCentralBX"      );
+	stdParameters.minNumberOfHits   = ps.getParameter<unsigned int>("minNumberOfHits"      );
 	stdParameters.requireBeamConstr = true;

```

Comment:
@perrotta I was thinking that the next update in the algorithm that can actually use the displaced and wide parameters. As of now, they are not used in the code at all as there have been no studies to understand if they are needed/or what to do. If you would like I can make them parameters, but I feel more odd about having parameters that don't change anything. I also feel a little odd about having the unused code there, but elected to keep it in for easier future development.

I can do the ParameterSetDescription